.github/workflows/testing-and-deployment.yml

```diff
@@ -205,15 +196,19 @@ jobs:
         run: python -c "import pyvista; print(pyvista.Report());"
 
       - name: Install Testing Requirements
-        run: pip install -r requirements_test.txt
+        run: pip install .[test]
 
       - name: Set up numpy
         if: ${{ matrix.numpy-version != 'latest' && matrix.numpy-version != 'nightly' }}
-        run: pip install 'numpy~=${{ matrix.numpy-version }}.0'
+        run: |
+          pip uninstall numpy -y
+          pip install 'numpy~=${{ matrix.numpy-version }}.0'
 
       - name: Limit Matplotlib for VTK<9.2.2
         if: ${{ matrix.vtk-version == '9.1' || matrix.vtk-version == '9.0.3' }}
-        run: pip install 'matplotlib<3.6'
+        run: |
+          pip uninstall matplotlib -y
```

Comment:
I'm curious, is this uninstall necessary? I would think/hope the following `install <x` line would handle uninstalling it

Reply:
I tried 
```
install matplotlib
install 'matplotlib<3.6'
```
and you are right, it works, so uninstall is not needed. This convention was copied from the `numpy` install a few lines above. Mayne that uninstall should also be removed.